### ISSUE TITLE
Hi, I Build SimplePlugin not Show in AbletonLive 10 Why?

### ISSUE THREAD

**Issue by crashangelbr** [ValdemarOrn/SharpSoundDevice#5]
Hi, I Building and use BridgeGenerator.exe to Generate Dlls and when a open Ableton in Plugins folder not show this plugin... Why???

I Use Ableton Live 10

**Comment by giavapps** [comment]
I know that this is an old post but maybe this can help other users.

I tested the whole process with a very basic code and just a few changes to the **SimplePlugin** example and works great.

Since I runned into the exact same problem described above I would like to give you guys some advices:

The first thing to note is that **Ableton Live 10** is a **64-bit** only **DAW** so better that you ensure that you are compiling a **64-bit** library.

The second thing you can do is **debug**!

As explained in the **Quickstart Guide** you can find the **Logs** at:

`C:\Users\<YourUserName>\AppData\Roaming\SharpSoundDevice\Logs`

Basically, if anything goes wrong when loading the plugin, the software doesn't list the plugin at all.

Hope this helps.

Greetings.
